MediaGalleryUi/view/adminhtml/templates/image_edit_details.phtml

```diff
@@ -48,12 +48,21 @@ use Magento\Backend\Block\Template;
                         "imageEditDetailsUrl": "<?= $escaper->escapeJs($block->getData('imageEditDetailsUrl')); ?>",
                         "saveDetailsUrl": "<?= $escaper->escapeJs($block->getData('saveDetailsUrl')); ?>",
                         "modalSelector": ".media-gallery-edit-image-details-modal",
-                        "mediaGridMessages": "standalone_media_gallery_listing.standalone_media_gallery_listing.messages"
+                        "mediaGridMessages": "standalone_media_gallery_listing.standalone_media_gallery_listing.messages",
+                        "viewConfig": [
+                            {
+                                "component": "Magento_Ui/js/form/element/ui-select",
+                                "name": "${ $.name }_keywords",
+                                "template": "ui/grid/filters/elements/ui-select",
+                                "disableLabel": true
+                            }
+                        ]
```

Comment:
Do we need to declare this viewConfig here? Can't we just do it inside the JS file?

Reply:
Thanks, generic component config was moved to the component defaults